src/lib/CMakeLists.txt

```diff
@@ -120,6 +120,8 @@ set(
     operators/maintenance/show_columns.hpp
     operators/maintenance/show_tables.cpp
     operators/maintenance/show_tables.hpp
+    operators/materialize.cpp
+    operators/materialize.hpp
```

Comment:
Files with that name already exists, which offers wide potential for confusion. Sort it out be renaming those in storage or these here, idc